src/blr/cli/tagbam.py

```diff
@@ -47,10 +48,16 @@ def main(args):
 
 
 def add_arguments(parser):
-    parser.add_argument("input_mapped_bam",
-                        help="BAM file with mapped reads which is to be tagged with barcode ids.")
-    parser.add_argument("output_tagged_bam",
-                        help="BAM file with barcode cluster id in the bc tag.")
+    parser.add_argument("input",
+                        help="SAM/BAM file with mapped reads which is to be tagged with barcode information. To read "
+                             "from stdin use '-'.")
+
+    parser.add_argument("-o", "--output", default="-",
+                        help="SAM/BAM file with barcodes information in the specified SAM tags. Default: write to "
+                             "stdout.")
```

Comment:
Since we already have the `get_output_mode` in `utils` which can discern file format from the name I think it's better if we instead assume output format from filenames.